src/main/java/org/jabref/model/entry/LinkedFile.java

```diff
@@ -220,6 +223,10 @@ public boolean isOnlineLink() {
         return isOnlineLink(link.get());
     }
 
+    public boolean isImage() {
+        return IMAGE_EXTENSIONS.stream().anyMatch(getFileType().toLowerCase()::contains);
```

Comment:
I've discovered that exclusively image files have the `String` "image" in their `fileTypeProperty` - should I just check for that `String` instead of using `HashSet` altogether?